manifests/service.pp

```diff
@@ -15,4 +15,11 @@
     }
   }
 
+  $use_servers = join($servers, " ")
+  if $autosync {
+    exec { "/usr/sbin/ntpdate -u -s ${use_servers}":
+     refreshonly => true
```

Comment:
Oh I found what triggers it: https://github.com/puppetlabs/puppetlabs-ntp/blob/3.3.0/manifests/init.pp#L60

(My question was because it is a `refreshonly` resource, but nothing notifies the resource directly.)